hypervisor/arch/x86/guest/instr_emul.c

```diff
@@ -1545,7 +1546,8 @@ vmm_emulate_instruction(struct vcpu *vcpu, uint64_t gpa, struct vie *vie,
 }
 
 int
-vie_alignment_check(uint8_t cpl, uint8_t size, uint64_t cr0, uint64_t rf, uint64_t gla)
+vie_alignment_check(uint8_t cpl, uint8_t size, uint64_t cr0, uint64_t rf,
+	uint64_t gla)
 {
```

Comment:
What is changed?

Reply:
What I changed here is to make sure that the line length is less than 80.